client/src/ui/Landingnavbar.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+import Image from "next/image"
+import Link from "next/link"
+import { Button, Typography } from "@mui/material";
+import "./styling/landingnavbar.css"
+import { customTheme } from "@/theme/theme";
+import {ThemeProvider} from "@mui/material";
+
+export default function Landingnavbar(){
+
+    return(
+    <nav className="navbar">
+        <div className="navbar-container">
+            <Link href='/' className="name-logo">
+                
+                <div className="logo">
+                    <Image className="logo-image"
+                        src="/rat_1.png"
+                        height={100}
+                        width={100}
+                        alt="Squeakpeek Logo"
+                        />
+                </div>
+                <Typography variant="h4">
+                    SqueakPeek
+                </Typography> 
+            </Link>
+            <div className="nav-menu">
+                <ul className="nav-links">
```

Comment:
Is there a reason why you're using a unordered list? If you used a div instead, you don't have to wrap the component into the li tags since the children are going to be wrapped in the Link component